.env.example

```diff
@@ -37,6 +37,7 @@ INGESTER_BIG_TABLE_CONFIG='{creds="/usr/src/app/creds.json", timeout=1000}'
 
 INGESTER_DISABLE_SYNCHRONIZER=true
 INGESTER_RUN_SEQUENCE_CONSISTENT_CHECKER=true
+INGESTER_RUN_FORK_CLEANER=false # Optional, required only if it needs to run fork cleaner, default is false. Unstable as it removes forked items, but also removes some valid leafs. Reccomended to use only! for testing purposes.
```

Comment:
Ultra nit-picky. I'd prefer the comment above the parameter as it it's kinda widely accepted. "Reccomended" typo also.